Makefile

```diff
@@ -200,3 +180,21 @@ $(NODE_MODULES_TEST): package.json
 	env -u NODE_ENV npm prune
 
 .PHONY: all clean install devel-install dist node-cache rpm check vm update-po
+
+# checkout common files from Cockpit repository required to build this project;
+# this has no API stability guarantee, so check out a stable tag when you start
+# a new project, use the latest release, and update it from time to time
+COCKPIT_REPO_FILES = \
+	pkg/lib \
+	test/common \
+	$(NULL)
+
+COCKPIT_REPO_URL = https://github.com/cockpit-project/cockpit.git
+COCKPIT_REPO_SHA = 4f8ad7314676b375fcca5c98a543ef5078cc1af5 # 267
```

Comment:
Bumping to ≥ 267 requires some adjustments first, see PR #672. Adding blocked for that.

Reply:
I dropped back down to 264-137-gd1a410cf9.  Is that okay?